docker/Dockerfile

```diff
@@ -308,7 +304,8 @@ RUN install-package \
     vim \
     nano \
     clinfo \
-    doxygen
+    doxygen \
+    zlib
```

Comment:
If you're installing zlib with arch instead of building it (which for this library is probably fine) you should make sure it's installed at the start of this file (before all the things that need it) rather than at the end.

That said I think it's included by default so this probably does nothing anway